packages/server/__tests__/db/seeds/fixtures.js

```diff
@@ -157,8 +157,14 @@ module.exports = {
 };
 
 module.exports.seed = async (knex) => {
-    const deletions = Object.values(TABLES).map((tableName) => knex(tableName).del());
-    await Promise.all(deletions);
+    // https://stackoverflow.com/a/36499676
+    const truncateStmt = `TRUNCATE TABLE ${Object.values(TABLES).join(', ')} RESTART IDENTITY CASCADE`;
+    await knex.raw(truncateStmt).catch(
```

Comment:
maybe it is better to drop the db? Not sure if that will work

Reply:
yeah can't drop the DB since it's actively in use when the drop cmd is called - tried it first